styles.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+  background-color: #F9F9F9;
+  }
+
+.nav-bar {
+  display: inline-flex;
+  justify-content: space-between;
+  width: 100%;
+  position: top;
+  align-items: center;
+  }
+
+  #nav1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  #nav2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  #search-bar {
+    width: 700px;
+  }
+
+  #search-button {
+    width:10%;
+  }
+
+#img-logo {
+  height: 60%;
+  margin-right: 10em;
+  width: 100px;
+  margin-left: 15px;
+}
+
+#create-img, #apps-img, #notifications-img {
+  width: 50px;
+  height: 60%;
+}
+
+section {
+  width: 100;
+}
+
+h1 {
+  font-size: 24px;
+  font-family: Roboto, Arial, sans-serif;
+  text-decoration: underline;
+}
+
+iframe {
+  width: 68%;
+  height: 400px;
+}
+
+ul {
+  width: 68%;
+  display: inline-flex;
+  align-items: center;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline-flex;
+  align-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  padding: 4px;
+  font-size: 14px;
+}
+
+.views {
+  margin-left: 20px;
+  justify-content: flex-end;
+}
+
+
+a {
+  text-decoration: none;
+  color: gray;
+  font-size: 16px;
+}
+
+#right-item {
+  float: right;
+}
+
+#subscribe-btn {
+  width: 110px;
+  height: 30px;
+  float: right;
+   
+}
+
+article {
+  width: 68%;
+}
+
+/*.
+  split {
+  height: 100%;
+  top: 35px;
+  position:absolute;
+  overflow-x: hidden;
+  padding-top: 40px;
+}
+*/
```

Comment:
Always delete commented code / block of codes. Please remove this block.
![1](https://user-images.githubusercontent.com/26856491/93934450-1797f380-fd1b-11ea-8831-4678d0e75fe8.png)